ipv-core/src/main/java/com/ibm/whc/deid/util/ContinentManager.java

```diff
@@ -27,7 +27,7 @@
 
   private static final long serialVersionUID = -610638379564157663L;
 
-  protected Map<String, List<Location>> continentListMap;
+  protected transient volatile Map<String, List<Location>> continentListMap;

```

Comment:
We could look at making Location Serializable and we could define the Map and List as actual types if that is impacting the scans.